src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -155,10 +155,10 @@ def test_generate_mlbf(self):
             assert key not in bfilter
         assert stats['mlbf_version'] == 2
         assert stats['mlbf_layers'] == 1
-        assert stats['mlbf_bits'] == 21608
+        assert stats['mlbf_bits'] == 2160
```

Comment:
fwiw, the bit counts and file sizes are decreasing because 0.4.0 reduced some of the minimum defaults and these are very small filters with only a few blocks.